Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -176,26 +182,18 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
         return editorScript
     }
 
-    // MARK: - Public API
-
-    // TODO: synchronize with the editor user-generated updates
-    // TODO: convert to a property?
-    public func setContent(_ content: String) {
-        _setContent(content)
+    private func evaluate(_ javascript: String, isCritical: Bool = false) {
+        webView.evaluateJavaScript(javascript) { [weak self] _, error in
+            guard let self, let error else { return }
+            self.handleError(error, isCritical: isCritical)
+        }
     }
 
-    private func _setContent(_ content: String) {
-        guard _isEditorRendered else { return }
-
-        let escapedString = content.addingPercentEncoding(withAllowedCharacters: .alphanumerics)!
+    public func setContent(_ newValue: String) {
+        let escapedString = newValue.addingPercentEncoding(withAllowedCharacters: .alphanumerics)!
         evaluate("editor.setContent(decodeURIComponent('\(escapedString)'));", isCritical: true)
```

Comment:
Not for this PR, but we can potentially create a simple wrapper to safely pass any types of arguments to JS. Something like

```swift
func callJSFunction(name: String, arguments: [AnyEncodeable]) {
  let args = JSONEncode().encode(arguments)
  evaluate("\(name)(...\(args))")
}